src/Stripe.ios.js

```diff
@@ -32,22 +37,27 @@ class Stripe {
       types.paymentRequestWithApplePayOptionsPropTypes,
       options, 'options', 'Stripe.paymentRequestWithApplePay'
     )
+    checkInit(this)
     return TPSStripeManager.paymentRequestWithApplePay(items, options)
   }
-  completeApplePayRequest = () => (
+  completeApplePayRequest = () => {
+    checkInit(this)
     TPSStripeManager.completeApplePayRequest()
```

Comment:
For now, If you've changed braces you should take care of promises. Cause every call to native modules returns a promise. You should return it from a method.
```js
return TPSStripeManager.completeApplePayRequest()
```